src/content/docs/devops.mdx

```diff
@@ -450,8 +481,12 @@ nginx -t
 nginx -s reload
 ```
 
+</Steps>
+
 ## Work on API Instances
 
+<Steps>
```

Comment:
Missing indentation.

Also it looks like two sections might have been joined in single `Steps`